ser/train.py

```diff
@@ -0,0 +1,64 @@
+import torch
+import torch.nn.functional as F
+
+def train_model(validation_dataloader, training_dataloader, model, optimizer, epochs, device, name):
+    # train
+    with open('./experiments/model_'+name+'.txt', 'w') as f:
```

Comment:
kinda hacky but neat - you get the whole log this way. You might like to play with something liek:

https://github.com/fossasia/visdom

Rather than printing you log a number to a visdom server and it makes a chart in real time. At the end you can save the charts in your results dir or dump results from them.